woocommerce/woocommerce-ios
# Do not load tax information if the order is not editable

## Description

This PR addresses this https://github.com/woocommerce/woocommerce-ios/pull/10703#issuecomment-1719859807 from @joshheald, where it's mentioned that we can change the tax rate even if the order is not editable, causing a different total.

It's worth mentioning that that was possible even before, by changing the customer address (this is what we do when changing the tax rate). Even with that, we don't want to make it so prominent by showing a row to edit tax rates.

## Testing instructions


1. Go to Orders
2. Tap on a completed order
3. Make sure that, unlike before, you cannot set a new tax rate to the order.

## Screenshots


https://github.com/woocommerce/woocommerce-ios/assets/1864060/0b7cdfaf-b1f9-4351-9387-64b5308e6d51

---
